src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/test_acr_commands.py

```diff
@@ -3,34 +3,104 @@
+        credential = self.cmd('acr credential renew -n {} -g {} --password-name {}'.format(
+            registry_name, resource_group, 'password2')).get_output_in_json()
+        renewed_username = credential['username']
+        renewed_password = credential['passwords'][0]['value']
+        renewed_password2 = credential['passwords'][1]['value']
+        assert renewed_username and renewed_password and renewed_password2
+        assert username == renewed_username
+        assert password != renewed_password
+        assert password2 != renewed_password2
+
+        # test acr storage account update
+        if storage_account_for_update is not None:
+            self.cmd('acr update -n {} -g {} --storage-account-name {}'.format(
+                registry_name, resource_group, storage_account_for_update),
+                checks=[self.check('name', registry_name),
+                        self.check('location', location)])
+
+        # test acr delete
+        self.cmd('acr delete -n {} -g {}'.format(registry_name, resource_group))
+
     def test_check_name_availability(self):
         # the chance of this randomly generated name has a duplication is rare
         name = self.create_random_name('clireg', 20)
-        self.cmd('acr check-name -n {}'.format(name), checks=JMESPathCheck('nameAvailable', True))
+        self.kwargs.update({
+            'name': name
+        })
+
+        self.cmd('acr check-name -n {name}', checks=[
+            self.check('nameAvailable', True)
+        ])
 
     @ResourceGroupPreparer()
     @StorageAccountPreparer(parameter_name='storage_account_for_update')
     def test_acr_create_with_new_storage(self, resource_group, resource_group_location,
                                          storage_account_for_update):
         registry_name = self.create_random_name('clireg', 20)
 
-        self.cmd('acr create -n {} -g {} -l {} --sku {} --deployment-name {}'.format(
-            registry_name,
-            resource_group,
-            resource_group_location,
-            'Classic',
-            'Microsoft.ContainerRegistry'),
-            checks=[JMESPathCheck('name', registry_name),
-                    JMESPathCheck('location', resource_group_location),
-                    JMESPathCheck('adminUserEnabled', False),
-                    JMESPathCheck('sku.name', 'Classic'),
-                    JMESPathCheck('sku.tier', 'Classic'),
-                    JMESPathCheck('provisioningState', 'Succeeded')])
+        self.kwargs.update({
+            'registry_name': registry_name,
+            'rg_loc': resource_group_location,
+            'sku': 'Classic',
+            'deployment_name': 'Microsoft.ContainerRegistry'
+        })
+
+        self.cmd('acr create -n {registry_name} -g {rg} -l {rg_loc} --sku {sku} --deployment-name {deployment_name}',
+                 checks=[self.check('name', self.kwargs['registry_name']),
+                         self.check('location', self.kwargs['rg_loc']),
```

Comment:
Instead of `self.kwargs['registry_name']` you can just put `'{registry_name}'.  That is why JMESPathCheck was replaced with the check method--it automatically runs an invisible format(**self.kwargs) on the pieces.

Reply:
done